src/sdk/common/ERC4337Utils.ts

```diff
@@ -18,6 +18,7 @@ export interface UserOperation {
   paymasterVerificationGasLimit?: BigNumberish
   paymasterPostOpGasLimit?: BigNumberish
   paymasterData?: BytesLike
+  paymasterAndData?: BytesLike
```

Comment:
pls remove `paymasterAndData`, it was removed in EPv7

Reply:
Changes done.